pom.xml

```diff
@@ -2558,6 +2660,26 @@
       </modules>
     </profile>
 
+    <!--
+      The cloud profile enables the cloud module.
+      It does not declare the hadoop-* artifacts which
+      the cloud module pulls in; these are delegated to
+      the hadoop-x.y protocols, so permitting different
+      hadoop versions to declare different include/exclude
+      rules (especially transient dependencies).
+
+      To use this profile, the hadoop-2.7 profile must also
```

Comment:
`hadoop-2.7` already also activates the `cloud` module ... but I see that the point is that `cloud` profile affects the assembly too.

OK, I like the flexibility though I guess I'm on the fence about keeping it separate. I'll put it this way: should one reasonably expect that something that builds around Hadoop 2.7 includes this support by default? if it's a kinda optional extra in Hadoop, then yes I can see how it makes sense to keep it optional here. BTW how about `hadoop-cloud` or something since this is narrowly about Hadoop deps?